package.json

```diff
@@ -43,6 +43,17 @@
     "@keystone-next/fields": "15.0.0",
     "@keystone-next/keystone": "27.0.1",
     "@next/bundle-analyzer": "12.1.0",
+    "@opentelemetry/api": "^1.0.4",
+    "@opentelemetry/auto-instrumentations-node": "^0.27.4",
+    "@opentelemetry/exporter-trace-otlp-grpc": "^0.27.0",
+    "@opentelemetry/id-generator-aws-xray": "^1.0.1",
+    "@opentelemetry/instrumentation-aws-sdk": "^0.5.1",
+    "@opentelemetry/node": "^0.24.0",
+    "@opentelemetry/propagator-aws-xray": "^1.0.1",
+    "@opentelemetry/resource-detector-aws": "^1.0.3",
+    "@opentelemetry/resources": "^1.0.1",
+    "@opentelemetry/sdk-trace-node": "^1.0.1",
+    "@opentelemetry/tracing": "^0.24.0",
```

Comment:
can we pin all of the above pls?

Reply:
YES this was just from testing -- thank you for the reminder!